tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -140,8 +140,14 @@ def set_storm_params(dut, fanout_info, fanout, peer_params):
     logger.info("Setting up storm params")
     pfc_queue_index = 4
     pfc_frames_count = 1000000
+    if dut.topo_type == 't2':
+        pfc_gen_file = 'pfc_gen_t2.py'
+        pfc_send_time = 8
+    else:
+        pfc_gen_file = 'pfc_gen.py'
     storm_handle = PFCStorm(dut, fanout_info, fanout, pfc_queue_idx=pfc_queue_index,
-                            pfc_frames_number=pfc_frames_count, peer_info=peer_params)
+                            pfc_frames_number=pfc_frames_count, pfc_gen_file=pfc_gen_file,
+                            pfc_send_period=pfc_send_time, peer_info=peer_params)
```

Comment:
pfc_send_time is not defined for non t2 topologies

Reply:
same comment as above

Reply:
Defined pfc_send_time for non-t2 cases as well. Please check.